action_scripts/convertToLowPoly.py

```diff
@@ -12,8 +14,8 @@ def convertToLowPoly(parameters, session_id):
        :param parameters: already parsed and checked parameters
        :param session_id: already validated session id of the user
     """
-    image_path = build_image_root_by_id(session_id)
     images = get_from_image_root(session_id)
+    image_format = images[0].split('.')[-1]
```

Comment:
Image_format -> Image_type?

Reply:
Every image has same type?

Reply:
According to the definition jpeg and png are image formats. Therefore I think the name image format is suitable.

Reply:
> Every image has same type?

Yes all images are from one session and have the same format.